.github/workflows/build.yml

```diff
@@ -80,9 +80,9 @@ jobs:
             architecture: "x64"
             vcvars_arch: "amd64"
             qt_ver: 6
-            qt_host: windows
-            qt_arch: ""
-            qt_version: "6.7.3"
+            qt_host: "windows_arm64"
```

Comment:
Why is `qt_host` for `amd64` `windows_arm64` ?

Reply:
wow, ok, that was fast

Reply:
I'm trying to fix workflow, seems they aqtinstall changed some stuff, and I am not good at ci stuff.

Reply:
> wow, ok, that was fast

![Screenshot 2024-10-10 040707](https://github.com/user-attachments/assets/bd7c1a5f-de64-41c2-b2f6-9be176f4589b)

Have you got any idea how to change that, I am out of ideas.